tests/integration/components/collection-submission-card/component-test.ts

```diff
@@ -62,10 +62,6 @@ module('Integration | Component | collection-submission-card', hooks => {
             'Request accepted 2 days ago by moderator',
             'Latest action shown',
         );
-        assert.dom('[data-test-submission-card-latest-action]').containsText(
```

Comment:
It may be worth trying to update this assertion to check something like `assert.dom('[data-test-review-action-comment]').containsText(. . .)`. It looks like `containsText` may not pick up nested text content?